x-pack/plugins/encrypted_saved_objects/server/saved_objects/encrypted_saved_objects_client_wrapper.ts

```diff
@@ -166,6 +168,13 @@ export class EncryptedSavedObjectsClientWrapper implements SavedObjectsClientCon
     return await this.options.baseClient.delete(type, id, options);
   }
 
+  public async bulkDelete(
+    objects: SavedObjectsBulkDeleteObject[],
+    options?: SavedObjectsBulkDeleteOptions
+  ) {
+    return await this.options.baseClient.bulkDelete(objects, options);
```

Comment:
Similar to `delete`, the bulk operation doesn't return any potentially sensitive data and can be passed directly to the base client.